tensorflow/core/kernels/mkl_slice_op.cc

```diff
@@ -193,12 +192,8 @@ class MklSlicePrimitive : public MklPrimitive {
     context_.dst_mem->set_data_handle(sliceParams.to->get_data_handle());
 
 #ifdef ENABLE_MKLDNN_V1
-    DCHECK_EQ(context_.slice_primitives.size(),
-              context_.slice_primitives_args.size());
-    for (size_t i = 0; i < context_.slice_primitives.size(); ++i) {
-      context_.slice_primitives.at(i).execute(
-          *context_.slice_stream, context_.slice_primitives_args.at(i));
-    }
+    execute_primitives(context_.slice_primitives, context_.slice_stream,
+                      context_.slice_primitives_args);
```

Comment:
This looks like it needs one more space. Please apply `clang-format`.
```suggestion
                       context_.slice_primitives_args);
```

Reply:
@penpornk Applied clang-format 8.0.1. Please check.
Thanks!